app/models/form/lettings/questions/irproduct_other.rb

```diff
@@ -5,6 +5,8 @@ def initialize(id, hsh, page)
     @check_answer_label = "Product name"
     @header = "Name of rent product"
     @type = "text"
-    @question_number = 6
+    @question_number = QUESION_NUMBER_FROM_YEAR.fetch(form.start_date.year, QUESION_NUMBER_FROM_YEAR.max_by { |k, _v| k }.last) if form.start_date.present?
```

Comment:
`if form.start_date.present?` is on setup section only, I think because some of the forms are still coming from JSON, so the form isn't set here (?) doesn't change things in terms of numbering